src/SDK/SmartDetectorsExtensionsPackage/SmartDetectorsExtensionsPackage.nuproj

```diff
@@ -40,11 +40,9 @@
     <ProjectReference Include="..\SmartDetectors.Extensions\SmartDetectors.Extensions.csproj">
       <PackageDirectory>lib</PackageDirectory>
     </ProjectReference>
+    <ProjectReference Include="..\SmartDetectorsPackage\SmartDetectorsPackage.nuproj" />
```

Comment:
Why this change?

Reply:
this lets nuproj know that it should take dependency on the SDK's package and not project
this way the SDK's DLL is not dropped to the nupkg of the extensions

---
In reply to: [244003117](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/40#discussion_r244003117) [](ancestors = 244003117)